src/onnxflow/justbuildit/hummingbird.py

```diff
@@ -118,6 +130,9 @@ def fire(self, state: build.State):
 
             Supported xgboost models:
               - xgboost.XGBClassifier
+
+            Supported lightgbm models:
+              - lightgbm.LGBMClassifier
```

Comment:
Should `LGBMRegressor` be in this list too? It is a supported instance type in `is_supported_lightgbm_model()`